operators/olm_v1/arch/olmv1-deppy.adoc

```diff
@@ -0,0 +1,86 @@
+:_content-type: ASSEMBLY
+[id="olmv1-deppy"]
+= Deppy (Technology Preview)
+include::_attributes/common-attributes.adoc[]
+:context: olmv1-deppy
+
+toc::[]
+
+{olmv1-first} uses the Deppy component and its resources to resolve dependencies.
+
+:FeatureName: {olmv1}
+include::snippets/technology-preview.adoc[]
+
+[id="olmv1-deppy-concepts"]
+== Concepts
+
+Deppy is a Kubernetes API for resolving constraints over catalogs of RukPak bundles. The initial goal of the project is to remove the dependency manager from the earlier OLM component and make it its own generic component.
+
+There are a set of expectations from the user that the package manager should never do the following:
+
+* Install a package whose dependencies cannot be fulfilled or that conflict with the dependencies of another package
+* Install a package whose constraints can not be met by the current set of installable packages
+* Update a package in a way that breaks another that depends on it
+
+The job of a resolver, given a set of entities to install and the constraints for each those entities, is to identify whether or not the entities are compatible.
+
+For Deppy, these entities can be default global constraints or things to be installed, such as Operators or plain bundles. The constraints for an entity define any dependencies the entity has or any other requirement for the entity, for example a version to pin the entity to.
```

Comment:
I feel like we should define entities first somehow. I got thrown in the first paragraph and wanted to s/packages/entities. Then I got to the second paragraph and realized the mistake. But this does get a little thick here. 

I guess I wonder how much we need these last two paragraphs right now, especially since customers won't be allowed to touch Deppy. And only internal customers, like pipeline builders, will need to understand the guts.

I also think the "not diagrams" do a pretty good job explaining what goes on under the hood.
WDYT?

Reply:
That's probably fair. The 3-bullet list right before these paragraphs sum up what is to be shown in the Example subsections that follow, too. These paragraphs are mostly just repeating the same idea(s).

Reply:
With those 2 paras gone, I think I'll also just change the "Entities" in the Examples to rally around "Packages".